x-pack/platform/plugins/shared/lens/public/app_plugin/shared/edit_on_the_fly/helpers.ts

```diff
@@ -150,3 +169,39 @@ export const getSuggestions = async (
   }
   return undefined;
 };
+
+export const injectESQLQueryIntoLensLayers = (
+  attributes: TypedLensSerializedState['attributes'],
```

Comment:
ℹ️ I would love to move this to the visualizations-utils package but the `TypedLensSerializedState` must move first and is a tricky one.